suite-native/app/.env.develop

```diff
@@ -1,3 +1,2 @@
-EXPO_PUBLIC_ENVIRONMENT=develop
```

Comment:
Not sure if we shouldn't keep it here if case we want to do develop build locally without EAS

Reply:
I mainly deleted it as it's slightly confusing that it's not used now (unless you build without EAS of course), but good point. I put it back.

Reply:
@Nodonisko Sorry, I ammended commits, so you can verify these changes in diff pls.